pepper-apis/src/main/java/org/broadinstitute/ddp/housekeeping/schedule/CheckKitsJob.java

```diff
@@ -81,12 +81,13 @@ public void execute(JobExecutionContext ctx) throws JobExecutionException {
             LOG.info("Running job {}", getKey());
             long start = Instant.now().toEpochMilli();
 
+            // DEPRECATED
             // We don't want to bombard DSM with kit status calls, so only call if time is up.
-            if (Instant.now().getEpochSecond() - lastStatusCheckEpochSecs > statusCheckSecs) {
-                TransactionWrapper.useTxn(TransactionWrapper.DB.APIS,
-                        handle -> kitCheckService.checkPendingKitStatuses(handle, dsmClient));
-                lastStatusCheckEpochSecs = Instant.now().getEpochSecond();
-            }
+            // if (Instant.now().getEpochSecond() - lastStatusCheckEpochSecs > statusCheckSecs) {
+            //     TransactionWrapper.useTxn(TransactionWrapper.DB.APIS,
+            //             handle -> kitCheckService.checkPendingKitStatuses(handle, dsmClient));
+            //     lastStatusCheckEpochSecs = Instant.now().getEpochSecond();
+            // }
```

Comment:
We're moving to hooking into the `TESTBOSTON_SENT` notifications instead of calling DSM directly.